commands/inventory/list_nodes.go

```diff
@@ -111,3 +125,7 @@ var (
 	ListNodes  = new(listNodeCommand)
 	ListNodesC = inventoryListC.Command("nodes", "Show nodes in inventory").Hide(hide)
 )
+
+func init() {
```

Comment:
I'm just wondering is it any convention/tradition to place `init()` above all other functions/methods?

Reply:
we usually place init below all functions.